index.html

```diff
@@ -2,30 +2,91 @@
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
 </head>
-<body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
 
-  Oh yeah, did we mention it's free?
-
-  Download on the App Store
-  Get it on Google Play
-
-  About Us
-  Support
-  Blog
-  Press
-  API
-  Jobs
-  Privacy
-  Terms
+<body>
+  <div>
+    <img class="background-img" src="/images/bgtile.png">
+  </div>
+  <div>
+    <div>
+      <main id="main-container">
+        <div class="left-side-container">
+          <img class="phone-image" src="/images/phones.png" alt="picture of phone">
+        </div>
+    
+        <div class="right-side-container">
+          <section class="right-top-container">
+            <img src="/images/brand.png" alt="instagram brand image">
+            <a href="#" class="myButton">
+              <img class ="house-icon" src="/images/home.png" alt="house icon">
+              Log in
+            </a>
+          </section>
+          
+          <section class="right-middle-container">
+            <h1>
+              Capture and Share <br>the World's Moments
+            </h1>
+    
+            <p>
+              Instagram is a <b>fast</b>, <b>beautiful</b> and <b>fun</b>  way to share your life<br> with friends and family.
+              <br>
```

Comment:
Don't use the tag <br>, it's better to add padding or margin between the paragraph.